src/overlays/actors/ovl_Demo_Gt/z_demo_gt.c

```diff
@@ -629,11 +629,11 @@ void DemoGt_Update8(DemoGt* this, PlayState* play) {
     func_8097EF00(this, play);
 }
 
-void DemoGt_Draw1(DemoGt* this, PlayState* play) {
+void DemoGt_Draw1(DemoGt* thisx, PlayState* play) {
     s32 pad;
     GraphicsContext* gfxCtx = play->state.gfxCtx;
     u32 gameplayFrames = play->gameplayFrames;
-    s16 pad2[2];
+    DemoGt* this = (DemoGt*)thisx;
```

Comment:
I would guess that all of these draw functions can (hopefully) be matched by being based `Actor*`. But understandable if you dont want to mess with that rn

Reply:
yeah, I already did it for another actor in this PR. Fortunately it turned out to be easy here